src/admin-views/tribe-options-general.php

```diff
@@ -270,6 +269,12 @@
 		return $fields;
 	} );
 } else {
+	if ( tribe_is_truthy( tribe_get_option( 'tribeDisableTribeBar', false ) ) ) {
+		$filter_activation[ 'attributes' ] = [ 'disabled' => 'disabled' ];
```

Comment:
:no_entry_sign: Array keys must NOT be surrounded by spaces if they only contain a string or an integer.

<sub>hash: 465f0c6dbb318be35165039e1e3e964c</sub>